hs_file_types/models/base.py

```diff
@@ -56,6 +56,13 @@ def get_metadata_elements(self):
         """
         return list(self.coverages.all())
 
+    def dict(self):
+        dict = {}
+        metadata = self.get_metadata_elements()
+        for element in metadata:
+            dict.update(element.dict)
+        return dict
+
```

Comment:
What would you suggest here?  dict function is producing a dictionary representation of the metadata element.  I guess it could be as_dict, but I don't see that being any more readable.